lib/containers/container_images.pm

```diff
@@ -66,7 +67,7 @@ sub build_and_run_image {
     assert_script_run "curl -f -v " . data_url('containers/index.html') . " > $dir/BuildTest/index.html";
 
     # At least on publiccloud, this image pull can take long and occasinally fails due to network issues
-    $builder->build($dir . "/BuildTest", "myapp", timeout => 600);
+    $builder->build($dir . "/BuildTest", "myapp", (timeout => is_x86_64 ? 600 : 1200));
```

Comment:
```suggestion
    $builder->build($dir . "/BuildTest", "myapp", timeout => is_x86_64 ? 600 : 1200);
```

Reply:
It's not needed, but it also doesn't harm :), as the parameters of a function are always just simple array.

Reply:
I'd like to keep it that way, unless there is a clear style guide suggestion otherwise

Reply:
I also kind of like it without (), but personal opinion too. Or if you wanna keep, at least moving the () to the actual operation, not before the variable name which looks a bit weird.. `timeout => (operation)`